src/background.js

```diff
@@ -176,20 +176,3 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
 });
 
 // at end of background.js
-chrome.commands.onCommand.addListener((command) => {
```

Comment:
remove this part of chode is necessary or by accident?

Reply:
If I don't remove that part of code the extension will refuse to open and show:
`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'onCommand')` as in #586 
That part is actually redundant because we don't need to listen to onCommand to handle media any more. The mediaSession api already cover that for us.